docker/conference/docker-compose.yml

```diff
@@ -35,6 +35,6 @@ services:
     ports:
       - "3004:3004"
     command: sh -c "service mongodb start && service rabbitmq-server start \
-                    && cd /owt-server/dist/bin && ./init-all.sh && ./start-all.sh && sleep infinity"
+                    && cd /home && ./startowt.sh"
     depends_on:
-      -  owt-server-conference-install
+            -  owt:run
```

Comment:
Sorry I don't know why I can't add suggestions for lines you didn't change, but you need to add the following

line 7:
dockerfile: Dockerfile
target: owt-run

line 40: should not be changed to owt:run, the above change will do what we want I believe.

Thanks

Reply:
I misunderstand the function yaml file provides before. I thought it was used to deploy the owt service before, so i simply modify the image to owt:run. After checking the yaml file, i found that it was used to generate different Docker images, will update the yaml file.